garden_ai/garden.py

```diff
@@ -1,26 +1,27 @@
+import json
 import logging
 import os
 import time
-import json
 from pathlib import Path
+from typing import List, Union
 
+import requests
 from globus_sdk import (
+    AuthAPIError,
+    AuthClient,
     GroupsClient,
-    SearchClient,
-    RefreshTokenAuthorizer,
     NativeAppAuthClient,
-    AuthClient,
-    AuthAPIError,
+    RefreshTokenAuthorizer,
+    SearchClient,
 )
-from typing import List
+from globus_sdk.scopes import ScopeBuilder
 from globus_sdk.tokenstorage import SimpleJSONFileAdapter
-
-from garden_ai.models import Garden
 from pydantic import ValidationError
 
+from garden_ai.models import Garden, Pipeline
 
 # garden-dev index
-GARDEN_INDEX_UUID = '58e4df29-4492-4e7d-9317-b27eba62a911'
+GARDEN_INDEX_UUID = "58e4df29-4492-4e7d-9317-b27eba62a911"
```

Comment:
Are all these little quote changes from a linter run or something?

Reply:
yeah black's default iirc is to never use single quotes unless replacing them risks changing the semantics